assets/src/scss/pages/_listing-page.scss

```diff
@@ -335,36 +335,42 @@
   }
 }
 
-.grid-view-toggle,
-.list-view-toggle {
+.layout-button-ctn {
   display: none;
 
   @include medium-and-up {
-    display: block;
     border-radius: 50%;
-    cursor: pointer;
-    background-color: transparent;
-    transition: background-color 150ms linear;
-    border: none;
-    padding: 2px;
-
-    .icon {
-      height: 40px;
-      width: 40px;
-      top: 0;
-    }
+    display: inline-block;
 
     &:hover {
       background-color: var(--grey-200);
     }
 
-    &:focus {
+    &:focus-within {
       background-color: var(--grey-200);
       @include focus-styles(50%);
     }
   }
 }
 
+.layout-toggle-grid,
```

Comment:
Can't we just use `layout-toggle` here? And then use `&.layout-toggle-grid` for example to add the correct image